client/src/storage/storage.ts

```diff
@@ -1,24 +1,20 @@
 import {IGenomeData} from "../genetics/genome"
 import {Hexalot} from "../island/hexalot"
-import { Island, IslandPattern } from "../island/island"
-import {Journey} from "../island/journey"
+import {Island, IslandData} from "../island/island"
+import {IJourneyData} from "../island/journey"
 
 export interface IStorage {
```

Comment:
Might want to rename this something like `IAppStorage` to disambiguate

Reply:
I don't see ambiguity yet.